pandas/compat/func_defaults.py

```diff
@@ -0,0 +1,69 @@
+"""
```

Comment:
move this to ``pandas/compat/numpy/function.py`` (and move ``pandas/compat/numpy_compat.py`` to ``pandas/compat/numpy/__init.__``